src/cpp/src/llm_pipeline_static.cpp

```diff
@@ -195,52 +199,119 @@ StaticLLMPipeline::StaticLLMPipeline(
-    ).create_infer_request();
-    m_kvcache_request = core.compile_model(
-        m_kvcache_model, device, generate_config
-    ).create_infer_request();
-    // (8) Initialize tensors
+    const auto use_blobs = pop_or_default(pipeline_config, "USE_BLOBS", false);
+    auto prefill_config = pop_or_default(pipeline_config, "PREFILL_CONFIG", get_default_prefill_config(use_blobs));
+    auto generate_config = pop_or_default(pipeline_config, "GENERATE_CONFIG", get_default_generate_config(use_blobs));
+
+    if (!use_blobs) {
+        // (1) Read the template model - this will be kvcache model
+        m_kvcache_model = core.read_model(path / "openvino_model.xml");
+        // (2) Expose KV-cache input and output layers from kvcache model
+        ov::pass::StatefulToStateless().run_on_model(m_kvcache_model);
+        // (3) Align u4 ZP constants
+        align_u4_zp_constants(m_kvcache_model);
+        // (4) Replace KV-tensors for the entire cache to tensors only for new token
+        m_kvcache_model = redirect_new_kv_to_output(m_kvcache_model);
+        // (5) Clone the model - this will be prefill
+        m_prefill_model = m_kvcache_model->clone();
+        m_prefill_model->set_friendly_name(m_kvcache_model->get_friendly_name() + "_prefill");
+        // (6) Reshape both models to static shape
+        const auto kMaxPromptLen = pop_or_default(pipeline_config, "MAX_PROMPT_LEN", 1024u);
+        const auto kMinResponseLen = pop_or_default(pipeline_config, "MIN_RESPONSE_LEN", 150u);
+        // FIXME For some models KV-cache dim != 2u
+        m_kvcache_desc = KVCacheDesc { kMaxPromptLen, kMaxPromptLen + kMinResponseLen, 0u, 2u };
+        reshape_to_static(m_prefill_model, m_kvcache_desc.max_prompt_size, m_kvcache_desc.max_prompt_size);
+        reshape_to_static(m_kvcache_model, 1u, m_kvcache_desc.total_size);
+        // (7) Compile both model
+        // FIXME: Drop CACHE_DIR option if NPUW is enabled
+        merge_config_with(prefill_config, pipeline_config);
+        merge_config_with(generate_config, pipeline_config);
+        drop_cache_dir(prefill_config);
+        drop_cache_dir(generate_config);
+        m_prefill_request = core.compile_model(
+            m_prefill_model, device, prefill_config
+        ).create_infer_request();
+        m_kvcache_request = core.compile_model(
+            m_kvcache_model, device, generate_config
+        ).create_infer_request();
+    } else {
+        auto import_blob = [this, 
+                            &path,
+                            &pipeline_config,
+                            &core,
+                            &device](const std::string& model_name,
+                                     ov::AnyMap& model_config) {
+            auto blob_path = pop_or_default(model_config, "BLOB_PATH", std::string{});
+
+            if (blob_path.empty()) {
+                blob_path = (path /
+                    (std::string("openvino_") + model_name + ".blob")).generic_string();
```

Comment:
I believe it should be just `string()`. Is everything all right with slashes?

Reply:
Fixed, thanks! As long as it works on Windows, I think everything is all right!